OpenIPSL/Electrical/Renewables/PSSE/PlantController/BaseClasses/BaseREPC.mo

```diff
@@ -0,0 +1,161 @@
+within OpenIPSL.Electrical.Renewables.PSSE.PlantController.BaseClasses;
+partial model BaseREPC
+  "Base class for the plant controller of the WECC generic renewable models"
+
+  import Complex;
+  import Modelica.ComplexMath.arg;
+  import Modelica.ComplexMath.real;
+  import Modelica.ComplexMath.imag;
+  import Modelica.ComplexMath.conj;
+  import Modelica.Blocks.Interfaces.*;
+  parameter OpenIPSL.Types.ApparentPower M_b=SysData.S_b "Machine base power" annotation (Dialog(group="Power flow data"));
+  extends OpenIPSL.Electrical.Essentials.pfComponent(
+    final enabledisplayPF=false,
+    final enablefn=false,
+    final enableV_b=false,
+    final enableangle_0=true,
+    final enablev_0=true,
+    final enableQ_0=true,
+    final enableP_0=true,
+    final enableS_b=true);
+
+  parameter Boolean vcflag=true "Reactive dropp (False) or line drop compensation (True)" annotation (Dialog(tab="Control"));
```

Comment:
I guess you mean "droop"?

Reply:
also use true and false, not True and False since that is not how it is typed in Modelica.

Reply:
Actually reading it again I'm now unsure if you meant droop or drop.